index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Lembani Sakala's personal portfolio website.">
+    <title>Lembani Sakala - Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+        integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header class="header">
+        <nav class="nav">
+            <a class="brand" href="#">Welcome</a>
+            <a class="menu fa fa-bars fa-xl"></a>
+        </nav>
+    </header>
+
+    <section class="headline">
+        <div class="headline-content">
+            <h1 class="intro-1">Hey there. I'm Lembani</h1>
+            <h2 class="intro-2">I'm a software developer</h2>
+        </div>
+        <p class="description">
+            I can help you build a product , feature or website. Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <ul class="social-icons">
+            <li class="social-icon"><a class="fa-brands fa-twitter fa-xl" href="#"></a></li>
+            <li class="social-icon"><a class="fa-brands fa-linkedin fa-xl" href="#"></a></li>
+            <li class="social-icon"><a class="fa-brands fa-medium fa-xl" href="#"></a></li>
```

Comment:
[REQUIRED] The background dosen't match [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
![image](https://user-images.githubusercontent.com/74744787/158792224-4884360f-fb29-432e-907b-8607f5c23287.png)
as the previous reviewer mentioned also
![image](https://user-images.githubusercontent.com/74744787/158792336-02ea9bdb-2ccd-4287-8f4c-e8bf24322a72.png)

Reply:
As you suggested, I have set the _background size_ to _contain_ so that it fits the viewport.
This is how it displays when I test it. 
<img width="766" alt="Screenshot 2022-03-17 at 1 21 06 PM" src="https://user-images.githubusercontent.com/30483991/158798537-a1a2c011-6d7c-4ee9-b1ad-d7e9c3b8de90.png">